### DISCUSSION TITLE
CORS Missing Allow Origin with useFetch

### DISCUSSION THREAD

**Discussion by krxdow** [nuxt/nuxt#19276] [Questions]
Hello
I'm trying to retrieve the source code of a web page, on the server side,
in order to operate on it

I use the useFetch method, but I have an error like

```javascript
<script setup>
const response = await useFetch('https://www.jeuxdemots.org/rezo-dump.php?gotermsubmit=Chercher&gotermrel=crocodile&rel=&relin=norelin')
 console.log(response)
</script>
```

but in the browser console, I have an error like

```
Blocking of a cross-origin request: the "Same Origin" policy does not allow viewing the remote resource located at 
{URL}
 Reason: "Access-Control-Allow-Origin" CORS header is missing. Status code: 200.
```

I looked at the [documentation](https://nuxt.com/docs/guide/concepts/rendering#route-rules)
and add this my configuration file

```javascript
export default defineNuxtConfig({

        routeRules : {
            '/_nuxt/**': { headers: { 'access-control-allow-methods': 'GET' } },
            '/api/v1/**': { cors: true},
        },
}
````
Anyone have an idea how to make it work

**Comment by dsfcoll** [comment]
Cors should be configured on API server you requesting. Or setup a proxy and communicate with that proxy.

**Reply by krxdow** [reply]
in fact it's good, it's the server side that I want to recover so I passed `ssr: true`
and it works
thank  you
